engine/common/system.c

```diff
@@ -129,6 +133,11 @@ const char *Sys_GetCurrentUser( void )
 
 	if( GetUserName( s_userName, &size ))
 		return s_userName;
+#elif XASH_PSVITA
+	static string username;
+	sceAppUtilSystemParamGetString( SCE_SYSTEM_PARAM_ID_USERNAME, username, sizeof( username ) - 1 );
+	if( username[0] )
```

Comment:
You can use COM_CheckStringEmpty macro here:
https://github.com/FWGS/xash3d-fwgs/blob/master/public/crtlib.h#L108=

Reply:
Done.